web/web-mock/web-mock-soap/src/main/java/com/castlemock/web/mock/soap/controller/mock/AbstractSoapServiceController.java

```diff
@@ -72,15 +73,7 @@
 import java.net.HttpURLConnection;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Date;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Random;
+import java.util.*;
```

Comment:
Minor thing, but it would be great if we avoid import the whole java.util.* and instead just import these classes that are required. You can update this in you IDE so that it doesn't use wildcard when importing.